app/models/manageiq/providers/vmware/cloud_manager/orchestration_template.rb

```diff
@@ -1,4 +1,6 @@
 class ManageIQ::Providers::Vmware::CloudManager::OrchestrationTemplate < OrchestrationTemplate
+  belongs_to :ext_management_system, :foreign_key => :ems_id, :inverse_of => :orchestration_templates
```

Comment:
We were setting the `ems_id` for these but didn't actually have the association.  It seems weird that these are being refreshed but not owned by an EMS for any other provider...need to look into this

Reply:
I remember the"not owned by an EMS" being intentional - something about the raw templates themselves are basically just text, so they SHA it and that's how they compare to other templates, but i don't remember the details...we'd have to dig into the history.

Reply:
https://github.com/ManageIQ/manageiq/pull/10244#discussion_r73436521

So the orchestration_templates that we pull down don't have the full text of the template and aren't useful by themselves, they have to be referenced on the target provider when provisioning.

Reply:
But they never added an association to the cloud manager, so not sure how they found these unless they were doing queries like `OrchestrationTemplate.where(:ems_id => ...)`

Reply:
If I recall, they do a query across all OrchestrationTemplates (perhaps by type)

Reply:
Okay, either way this was setting the ems_id before so I think we need to maintain that.